cmd/download.go

```diff
@@ -66,3 +72,10 @@ func writeFile(path, contents string) error {
 	}
 	return ioutil.WriteFile(path, []byte(contents), 0644)
 }
+
+// Parses Submission ID from url strings
+// e.g "http://exercism.io/submissions/<SUBMISSION_ID>" --> <SUBMISSION_ID>
+func parseSubmissionID(submission string) string {
```

Comment:
How about `submissionKeyFromURL` as a name for this? parseSubmissionID sounds like it's parsing the ID, which it's not. A name starting in _parse_ tends to focus on what the function is doing rather than what you get out of it, which can be harder to understand.

The function doesn't need to be commented, since it's not exported.